src/primer.tex

```diff
@@ -124,7 +124,7 @@ \section{Bitcoin and Other Decentralized Protocols}
 The core Internet protocols are decentralized.
 Many other Internet-based protocols are similarly decentralized,
```

Comment:
well, @carlysheridan says lowercase, so lowercase it is. Consistency is the important thing here